week2/Task5/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿// 5. Better printout
+
+var time = new JupiterTime { Hours = 2, Minutes = 70 };
+Console.WriteLine(time);
+
+time.AddHours(1);
+Console.WriteLine(time);
+time.AddMinutes(65);
+Console.WriteLine(time);
+class JupiterTime
+{
+    public int Hours { get; set; }
+    public int Minutes { get; set; }
+
```

Comment:
here you can keep the constructor that you had in the previous steps, with the logic inside 
` public JupiterTime(int hours, int minutes){
        Hours = hours;
        Minutes=minutes;

        if(Minutes>59){
            Hours += minutes/60;
            Minutes = minutes % 60;
        }
    }`
    
    and then the ToString override can be just the printing
    ` public override string ToString()
    {
        return "Time: " + Hours + ":" + Minutes;
    }`

Reply:
Thanks.Done